### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mfojtik**
This gives us a starting point for future updates.

/cc @deads2k 
/cc @sttts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,71 @@
...
+* `cluster-config-v1` *configmap* in *kube-system* namespace
+  - The observed configmap `install-config` is decoded and the `networking.podCIDR` and `networking.serviceCIDR` is extracted and used as input for `admissionPluginConfig.openshift.io/RestrictedEndpointsAdmission
+.configuration.restrictedCIDRs` and `servicesSubnet`
+
+
+The configuration for the Kubernetes API server is the result of merging:
+
+* a [default config](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/v3.11.0/kube-apiserver/defaultconfig.yaml)
+* observed config (compare observed values above)
+  - `spec.spec.unsupportedConfigOverrides` from the `kubeapiserveroperatorconfig`.
```
Comment (sttts) [inline README.md]: wrong indention

**Comment by mfojtik** [comment]
@sttts indentation fixed.

**Comment by sttts** [comment]
/lgtm

We can iterate on it.

**Comment by deads2k** [comment]
I doubt the readme broke CI.  Merging

**Pull request merged**